vault/data_source_pki_secret_backend_issuer_test.go

```diff
@@ -5,13 +5,13 @@ package vault
 
 import (
 	"fmt"
+	"github.com/hashicorp/terraform-provider-vault/internal/provider"
```

Comment:
nit: import formatting

Reply:
Hmm, `make fmt` is not changing it (with go 1.23.5).

Reply:
Yeah it doesn't do imports oddly enough, goimports will fix it but there are other files with issues as well in the repo.